File: js/task2/S2_ARR_06.js
function activeUserNames(users) {
  return users
    .filter(u => u.active)
    .map(u => u.name.toUpperCase())
    .sort();
}

const users = [
  { id: 1, name: "Ana", active: true },
  { id: 2, name: "Luis", active: false },
  { id: 3, name: "Pedro", active: true }
];

console.log(activeUserNames(users)); // ["ANA","PEDRO"]

File: python/task2/S2_ARR_06.py
users = [
    {"id":1,"name":"Ana","active":True},
    {"id":2,"name":"Bob","active":False},
    {"id":3,"name":"Carla","active":True}
]

result = sorted([u["name"].upper() for u in users if u["active"]])
print(result)
